packages/boba/gateway/src/components/availableBridges/availableBridges.js

```diff
@@ -37,17 +38,20 @@ function AvailableBridges({ token = null, walletAddress = "" }) {
       </Typography>
     </S.LabelContainer>
     <S.Wrapper>
-      <S.BridgeContent key={'banxa'}>
-        <Link color="inherit"
-          variant="body2"
-          href={banxaUrl()}
-          target="_blank"
-          rel="noopener noreferrer"
-          sx={{ textDecoration: 'none' }}
-        >
-          <Typography variant="body1" component="span" my={1}> Banxa</Typography>
-        </Link>
-      </S.BridgeContent>
+      {currentNetwork === 'ETHEREUM' && isAvailableOnBanxa && (
```

Comment:
import network from constants.
```suggestion
      {currentNetwork === NETWORK.ETHEREUM && isAvailableOnBanxa && (
```